### PR TITLE
fix: roll webpack-deep-scope-plugin back

### PR DESCRIPTION
**Pull request by EvanBacon**
Related https://github.com/expo/expo-cli/issues/614

Deep scope analysis was rewritten and only the patch version was bumped. I suspect this could be what broke web builds.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1374,47 +1309,46 @@
...
 
 "@lerna/add@^3.4.1":
-  version "3.13.1"
-  resolved "https://registry.yarnpkg.com/@lerna/add/-/add-3.13.1.tgz#2cd7838857edb3b43ed73e3c21f69a20beb9b702"
-  integrity sha512-cXk42YbuhzEnADCK8Qte5laC9Qo03eJLVnr0qKY85jQUM/T4URe3IIUemqpg0CpVATrB+Vz+iNdeqw9ng1iALw==
-  dependencies:
-    "@lerna/bootstrap" "3.13.1"
-    "@lerna/command" "3.13.1"
-    "@lerna/filter-options" "3.13.0"
+  version "3.15.0"
```
Comment (fson) [inline yarn.lock]: These changes seem unrelated to this pull request, I wonder where they came from?

**Comment by EvanBacon** [comment]
Related https://forums.expo.io/t/expo-web-cli-is-broken/24721
